include/utils.h

```diff
@@ -5,44 +5,38 @@
 #include "assert.h"
 
 #ifdef _DEBUG
-constexpr bool debug_mode = true;
```

Comment:
👎 `constexpr` is available in C++11.

Reply:
Good point, reverted with 030be213477218892a8442cf622e80cf71d962be

Reply:
Just fyi constexpr if is not available until C++ 17: https://en.cppreference.com/w/cpp/language/if. Thus there's no point in making the bool a constexpr here.